aperture.go

```diff
@@ -612,37 +612,34 @@ func getTLSConfig(serverName, baseDir string, autoCert bool) (
 // specified in the config. Onion services will be created over which the proxy
 // can be reached at.
 func initTorListener(cfg *Config, etcd *clientv3.Client) (*tor.Controller, error) {
+	// Determine tor onion type based on config flags.
+	var onionType tor.OnionType
+	if cfg.Tor.V2 {
```

Comment:
See my previous comment about V2 being deprecated. Would clean up the config and code quite a bit if we removed it.

Reply:
Will remove tor v2 support, following a suggestion from your other comment.